apps/spruce/src/pages/projectSettings/tabs/GithubCommitQueueTab/getFormSchema.tsx

```diff
@@ -196,6 +197,10 @@ export const getFormSchema = (
         mergeQueue: {
           type: "object" as "object",
           title: "Merge Queue",
+          ...(projectType === ProjectType.Repo && {
+            description:
+              "If enabled, these settings can only apply to one branch project that also has this feature enabled (and does not apply to untracked branches).",
```

Comment:
[nit] This may be slightly clearer?
```suggestion
              "If enabled, these settings can only apply to one branch project that also has this feature enabled. They do not apply to untracked branches.",
```

Reply:
haha this is what i had originally and then readign it back i thought it was more confusing. I'm fine with this though.